src/config/loading/schema.rs

```diff
@@ -0,0 +1,80 @@
+use super::ConfigBuilder;
+
+const SECRET_AND_VARIABLES_REGEX: &str = "(:?\\$\\{.+\\})";
```

Comment:
Does this actually detect usage of the secrets backend? 🤔

Reply:
good point, I'll update that 👍